website/source/docs/agent/http/agent.html.markdown

```diff
@@ -196,6 +198,18 @@ It returns a JSON body like this:
 }
 ```
 
+### <a name="agent_reload"></a> /v1/agent/reload
+
+Added in Consul 0.7.2, this endpoint is hit with a PUT and is used to instruct
```

Comment:
Some dude went through and put all the HTTP verbs in backticks so we should keep that style here. Looks nicer.